nav2_controller/doc/README.md

```diff
@@ -0,0 +1,69 @@
+# Local Planner
+
+## Local Planner under ROS 1
+
+Under ROS 1, the navigation stack provides a `BaseLocalPlanner` interface
+used by `MoveBase`. The `BaseLocalPlanner` component is expected to take a path and current position and produce a command velocity. The most commonly used implementation of the local planner uses the DWA algorithm.
+
+The navigation repository provides two implementations of the DWA algorithm: `base_local_planner` and `dwa_local_planner`. In addition, there is another DWA based planner in the Robot Navigation repository called DWB. The `dwa_local_planner` was meant to replace `base_local_planner` and DWB was meant to replace both the others.
```

Comment:
it might be worth noting that base_local_planner also implements the "trajectory rollout" method (not just a hacky version of DWA). Trajectory Rollout is still in use on a number of robots out there, because it works quite a bit better for robots with low acceleration limits.

Reply:
Done